business_requirement/models/business.py

```diff
@@ -110,7 +110,9 @@ class BusinessRequirement(models.Model):
     )
     priority = fields.Selection(
         [('0', 'Low'), ('1', 'Normal'), ('2', 'High')],
-        'Priority', select=True
+        'Priority',
+        required=True,
```

Comment:
not sure you need the required here as this is a selection (nevertheless no harm anyway)
LGTM